### PR TITLE
Add iSCSI

### PR DESCRIPTION
**Pull request by mjg59**

### CODE REVIEW

File: sys-block/open-iscsi/files/iscsid.service [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Open-iSCSI
+Documentation=man:iscsid(8) man:iscsiuio(8) man:iscsiadm(8)
+After=network.target NetworkManager-wait-online.service iscsiuio.service tgtd.service targetcli.service
```
Comment (mischief) [inline sys-block/open-iscsi/files/iscsid.service]: can we get rid of `NetworkManager-wait-online.service` here? does open-iscsi ship tgtd or targetcli?

File: sys-fs/multipath-tools/files/multipath.rc [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/sbin/runscript
```
Comment (mischief) [inline sys-fs/multipath-tools/files/multipath.rc]: drop this file

File: sys-fs/multipath-tools/files/rc-multipathd [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/sbin/runscript
```
Comment (mischief) [inline sys-fs/multipath-tools/files/rc-multipathd]: drop this file

File: sys-fs/multipath-tools/multipath-tools-0.5.0-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+		libudevdir='${prefix}'/"${udevdir}" \
+		install
+
+	insinto /usr/share/multipath-tools
+	newins "${S}"/multipath.conf.annotated multipath.conf
+	# /etc/udev is reserved for user modified rules!
+	mv "${D}"/etc/udev/rules.d "${D}/${udevdir}"/ || die
+	fperms 644 "${udevdir}"/rules.d/66-kpartx.rules
+	newinitd "${FILESDIR}"/rc-multipathd multipathd
+	newinitd "${FILESDIR}"/multipath.rc multipath
```
Comment (mischief) [inline sys-fs/multipath-tools/multipath-tools-0.5.0-r1.ebuild]: drop newinitd's

File: sys-block/open-iscsi/files/iscsid.service (hunk 1 above)
Comment (mjg59) [inline sys-block/open-iscsi/files/iscsid.service]: We can, but we're divergent from upstream in that case - After= is only relevant if those other services exist and are enabled, it'll still start fine without them.

**Comment by marineam** [comment]
LGTM

**Pull request merged**